addon/components/paper-sidenav-inner.js

```diff
@@ -49,7 +47,8 @@ export default Component.extend(TransitionMixin, {
 
   didInsertElement() {
     this._super(...arguments);
-    $(window).on(`resize.${this.elementId}`, run.bind(this, 'updateLockedOpen'));
+    this._updateOnResize = run.bind(this, 'updateLockedOpen')
```

Comment:
Need `;` here.

Reply:
Can also use `run.bind(this, this.updateLockedOpen)` instead of the "magic" string.